### ISSUE TITLE
[Not ready for Triage/Review Meeting][Accessibility] [A11yMAS][Keyboard Navigation] [MAS2.1.1][HighImpact][ “Switch to the snippet you are using” is not accessible by Keyboard.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#723]
**User Experience:**
User who rely on keyboard will not be able to access those controls if they are not accessible by keyboard which may lead for external help.

**Test Environment:**
OS Build: 16299.19
Excel version:1712(8808.1000)
Prerequisites:
Add script lab “Add ON” In Excel 

**Attachments:**

**Repro Steps:**

1. Open Script Lab in Excel
2. Navigate to Run button and activate it.
3. Navigate to Run pane Using F6 key
4. Navigate to “Switch to the snippet you are using” in Run pane using Tab key.
5. Verify whether “Switch to the snippet you are using” is accessible or not.

**Current Behavior:**
“Switch to the snippet you are using” is not accessible by Keyboard.

**Expected Behavior:**
“Switch to the snippet you are using” should be accessible by Keyboard. So that keyboard user can be able to access all controls without any problem.

**Note:**
1.Controls(Firebug, Minimize/Maximize, Dom, Console) in Firebug section are not accessible.
2.All interactive controls in Back Pane and tool tip about the updated code are not accessible.

**MAS Reference:**
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.1.1
